### PR TITLE
fix: accounts in cross-chains.

### PR DESCRIPTION
**Pull request by web3max**

### CODE REVIEW

**Review by vecpeng — APPROVED** [review]

File: src/constant/index.ts [hunk 1]
```diff
@@ -30,3 +30,5 @@ export enum DigitalEmojiUnifiedMap {
   '8⃣' = '8️⃣',
   '9⃣' = '9️⃣'
 }
+
+export const CrossEthContract = '0x60eB332Bd4A0E2a9eEB3212cFdD6Ef03Ce4CB3b5'
```
Comment (zgayjjf) [inline src/constant/index.ts:34]: Some days, we can move all these constants to dotbit.js and separate them into a standalone package, named `@dotbit/constants`.

File: src/hooks/useAccount.ts [hunk 2]
```diff
@@ -39,33 +39,36 @@ const defaultAccount = {
...
+      rawRecords = await dotbit.records(account.value.account)
     }
     catch (err) {
       // @ts-expect-error
-      if (err.code === 'UnregisteredAccount') {
+      if (err.code === 20007) {
         account.value.status = AccountStatus.unregistered
       }
+      // @ts-expect-error
+      else if (err.code === 20008) {
```
Comment (zgayjjf) [inline src/hooks/useAccount.ts:69]: This can be added to BitIndexerErrorCode @vecpeng

File: src/hooks/useAccount.ts [hunk 3]
```diff
@@ -39,33 +39,36 @@ const defaultAccount = {
...
-      // @ts-expect-error
-      accountData = await das.account(account.value.account)
-      rawRecords = await das.records(account.value.account)
+      accountData = await dotbit.accountInfo(account.value.account)
+      rawRecords = await dotbit.records(account.value.account)
     }
     catch (err) {
       // @ts-expect-error
-      if (err.code === 'UnregisteredAccount') {
+      if (err.code === 20007) {
```
Comment (zgayjjf) [inline src/hooks/useAccount.ts:65]: This can use BitIndexerErrorCode.AccountNotExist later.

File: src/hooks/useAccount.ts [hunk 4]
```diff
@@ -74,43 +77,33 @@ export function useAccount (resolveResult: ResolveResult): {account: Ref<Account
...
     const descriptionRecord = profiles.find(record => record.key === 'profile.description')
     const welcomeRecord = customs.find(record => record.key === 'custom_key.bitcc_welcome')
     const themeRecord = customs.find(record => record.key === 'custom_key.bitcc_theme')
     const redirectRecord = customs.find(record => record.key === 'custom_key.bitcc_redirect')
     const backgroundImageRecord = customs.find(record => record.key === 'custom_key.bitcc_background_image')
 
-    const avatar = await das.getAvatar(account.value.account)
-
+    // const avatar = await dotbit.avatar(account.value.account)
+    const avatar = profiles.find(record => record.key === 'profile.avatar')
```
Comment (zgayjjf) [inline src/hooks/useAccount.ts:106]: We can use `dotbti.avatar` once `@dotbit/plugin-avatar` is released. @vecpeng

File: src/modules/tools.ts [hunk 5]
```diff
@@ -83,3 +87,18 @@ export function digitalEmojiUnifiedHandle (str: string): string {
   })
   return list.join('')
 }
+
+/**
+ * the account is converted to an NFT ID.
+ * @param account
+ */
+export function dotbitToNftTokenId (account: string): string {
```
Comment (zgayjjf) [inline src/modules/tools.ts:95]: Add a `accountTokenId` in dotbit.js, along side with `acountIdHex`.
@vecpeng

**Review by zgayjjf — APPROVED** [review]

**Pull request merged**